todoList/src/todoListUI.js

```diff
@@ -0,0 +1,159 @@
+module.exports = class {
+  createElementWithClass(tag, ...classes) {
+    const element = document.createElement(tag);
+
+    classes.forEach((cls) => {
+      element.classList.add(cls);
+    });
+
+    return element;
+  }
+
+  initTodoHeaderUI() {
+    const todoHeader = this.createElementWithClass('div', 'todo-header');
+
+    todoHeader.innerHTML = `
+    <div class="left-side"></div>
+    <div class="todo-contents"></div>
+    `;
+
+    return todoHeader;
+  }
+
+  initTodoInputBoxUI() {
+    const todoInputBox = this.createElementWithClass('div', 'todo-input-box');
+
+    const leftSide = this.createElementWithClass('div', 'left-side');
+    todoInputBox.appendChild(leftSide);
+
+    const addBtn = this.createElementWithClass('input', 'todo-add-btn', 'todo-btn');
+    addBtn.type = 'button';
+    addBtn.value = '⋁';
+    addBtn.name = 'add';
```

Comment:
이 부분을 보니 `createElementWithClass`의 개선점이 보이네요 ㅎㅎ
엘리먼트의 속성들도 옵션으로 받아서 처리해주면 좋을 것 같아요.

```suggestion
    const addBtn = this.createElementWithClass(['input', 'todo-add-btn', 'todo-btn'], {
        type: 'button',
        value: '⋁',
        name: 'add'
    });
```

Reply:
그리고 코드를 보니 `name`은 커스텀 프로퍼티로 보이더라고요. 버튼의 이벤트를 바인딩 할 때 타입처럼 쓰이고있는데,
`data-type`정도로 커스텀 데이터 프로퍼티임을 알 수 있게 명시적으로 사용하는게 좋을 것 같습니다.

Reply:
`createElementWithClass` 함수를 요소 타입과, 클래스 명의 배열, 속성 값 객체를 가지는 형태로 변경했습니다.
해당 함수가 더 이상 클래스만을 포함한 엘리먼트를 생성하지 않기 때문에 `createElement`로 함수 명을 변경했습니다.